modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -1858,8 +1851,8 @@ input ObservationPropertiesInput {
   # The scienceRequirements defaults to spectroscopy if not specified on creation, and may be edited but not deleted
   scienceRequirements: ScienceRequirementsInput
 
-  # The scienceMode describes the chosen observing mode and instrument, is optional and may be deleted
-  scienceMode: ScienceModeInput
```

Comment:
`scienceMode` is also used for the choice between imaging and spectroscopy.  I wanted to remove the ambiguity so I renamed this one to "observing mode" which I think might be more appropriate anyway.

Reply:
Seems good